File: src/main/java/nl/dannyjelsma/soundboardbot/commands/music/LoopCommand.java
package nl.dannyjelsma.soundboardbot.commands.music;

import com.jagrosh.jdautilities.command.Command;
import com.jagrosh.jdautilities.command.CommandEvent;
import nl.dannyjelsma.soundboardbot.audio.SoundManager;

public class LoopCommand extends Command {

    public LoopCommand() {
        name = "loop";
    }

    @Override
    protected void execute(CommandEvent event) {

        boolean currentLoop = SoundManager.getInstance().shouldLoop();

        SoundManager.getInstance().setLoop(!currentLoop);
        event.getChannel().sendMessage("Set loop to **" + !currentLoop + "**.").queue();
    }
}

File: src/main/java/nl/dannyjelsma/soundboardbot/commands/music/LoopCommand.kt
package nl.dannyjelsma.soundboardbot.commands.music

import com.jagrosh.jdautilities.command.Command
import com.jagrosh.jdautilities.command.CommandEvent
import nl.dannyjelsma.soundboardbot.audio.SoundManager

class LoopCommand : Command() {
    init {
        name = "loop"
    }

    override fun execute(event: CommandEvent) {
        val currentLoop: Boolean = SoundManager.shouldLoop()

        SoundManager.setLoop(!currentLoop)
        event.channel.sendMessage("Set loop to **" + !currentLoop + "**.").queue()
    }
}